src/vcpkg/vcpkgcmdarguments.cpp

```diff
@@ -86,7 +86,7 @@ namespace
 
             if (switch_.helpmsg)
             {
-                if (cmd_parser.parse_switch(name, tag, parse_result, switch_.helpmsg()) && parse_result)
+                if (cmd_parser.parse_switch(name, tag, parse_result, switch_.helpmsg.format()) && parse_result)
```

Comment:
(Sorry for not doing the investigation myself)

How does this avoid crashing for switches that have default constructed help (e.g. remove's `OPTION_PURGE`)?

Reply:
The `if` on like 87 will be false because `operator bool` on that thing returns false if there's nothing inside